seqsign/sequence_signature.py

```diff
@@ -1283,7 +1283,7 @@ def score_protein_class(self, pclass_slug='001', signprot=False):
             protein_scores[pcf] = (score, nscore)
             protein_signature_match[pcf] = signature_match
             p_end = time.time()
-            print("Time elapsed for {}: ".format(pcf.protein.entry_name), p_end - p_start)
+            # print("Time elapsed for {}: ".format(pcf.protein.entry_name), p_end - p_start)
```

Comment:
Codacy highlighted that by commenting this you can also comment the p_start and p_end (sa